recipe/build.sh

```diff
@@ -1,15 +1,40 @@
 #!/bin/bash
+echo "Building ${PKG_NAME}."
 
-#cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release     \
-#      -DCMAKE_INSTALL_PREFIX=$PREFIX \
-#      -DCMAKE_INSTALL_LIBDIR=lib     \
-#      -DCMAKE_CXX_FLAGS=-O2     \
-#      -DQL_BUILD_EXAMPLES=OFF   \
-#      -DQL_BUILD_BENCHMARK=OFF   \
-#      $SRC_DIR
+# Isolate the build.
+mkdir -p build
+cd build || exit 1
 
+export CXXFLAGS="${CXXFLAGS} -O2 -g0 -fno-fast-math -Wall -Wno-unknown-pragmas -Werror"
 
-./autogen.sh
-./configure --disable-static CXXFLAGS="-O2 -g0 -Wall -Wno-unknown-pragmas -Werror" --prefix $PREFIX
-make
-make install
+# TODO: Enable DQL_BUILD_TEST_SUITE with the next release. See comments below.
+cmake .. ${CMAKE_ARGS} \
+    -GNinja \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_INSTALL_LIBDIR=lib     \
+    -DCMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS" \
+    -DQL_BUILD_EXAMPLES=OFF   \
+    -DQL_BUILD_TEST_SUITE=OFF \
+    -DQL_BUILD_BENCHMARK=OFF
+
+# Build.
+echo "Building..."
+ninja -j${CPU_COUNT} || exit 1
+
+# Perform tests.
```

Comment:
This seems a little cryptic for someone in the future. Could be useful to say which of the below is logs with a bit more description and which should be uncommented.

Reply:
I clarified a comment